Uplift/Views/Onboarding/CreateProfileView.swift

```diff
@@ -0,0 +1,184 @@
+                readyGreeting
+                getStartedButton
+                    .padding(.top, 32)
+            } else {
+                Spacer(minLength: 200)
+                nextLabel
+            }
+
+            Spacer()
+        }
+        .padding(.vertical, 40)
+        .photosPicker(
+            isPresented: $isShowingImagePicker,
+            selection: $profileItem,
+            matching: .images,
+            photoLibrary: .shared()
+        )
+        .onChange(of: profileItem) { newItem in
+            Task {
+                if let newItem = newItem {
+                    if let data = try? await newItem.loadTransferable(type: Data.self),
+                       let image = UIImage(data: data) {
+                        DispatchQueue.main.async {
+                            self.profileImage = image
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private var allChecked: Bool {
+        isCheckedTerms && isCheckedData && isCheckedLocation
+    }
+
+    private var cameraMiniImage: some View {
+        Constants.Images.cameraMini
+            .resizable()
+            .frame(width: 24, height: 20)
+    }
+
+    private var cameraMiniButton: some View {
+        Button {
+            isShowingImagePicker = true
+        } label: {
+            Circle()
+                .fill(Constants.Colors.white)
+                .frame(width: 40, height: 40)
+                .overlay(cameraMiniImage)
+                .upliftShadow(Constants.Shadows.smallLight)
+        }
+        .offset(x: 60, y: 60)
+    }
+
+    private var cameraPlaceholder: some View {
+        ZStack {
+            if let profileImage = profileImage {
+                Image(uiImage: profileImage)
+                    .resizable()
+                    .scaledToFill()
+                    .frame(width: 161, height: 161)
+                    .clipShape(Circle())
+                    .overlay(Circle().stroke(Constants.Colors.white, lineWidth: 6))
+                    .upliftShadow(Constants.Shadows.smallLight)
+                cameraMiniButton
+            } else {
+                Constants.Images.camera
+                    .resizable()
+                    .frame(width: 161, height: 161)
+            }
+        }
+    }
+
+    private var cameraPlaceholderButton: some View {
+        Button {
+            isShowingImagePicker = true
+        } label: {
+            cameraPlaceholder
+        }
+    }
+
+    private var nextLabel: some View {
+        Text("Next")
+            .font(Constants.Fonts.h2)
+            .foregroundColor(Constants.Colors.black)
+            .padding(.horizontal, 52)
+            .padding(.vertical, 12)
+            .background(Constants.Colors.gray03)
+            .cornerRadius(38)
+            .upliftShadow(Constants.Shadows.smallLight)
+    }
+
+    private var getStartedButton: some View {
+        Button {
+            mainViewModel.displayMainView = true
```

Comment:
Maybe animate this, so wrap this in a `withAnimation { ... }` and add a `.animation` modifier where this view is updated (probably in the MainView). I would use a fade in with opacity